pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -640,9 +640,10 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 			//FIXME: gracePeriod is nil
 			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 			if err != nil {
-				return err
```

Comment:
These REALLY shouldn't be removed... If it errors out during deployment Kompose should return an err. It's not good Go practice to simply `log.Error(err)`.

If we're retrying for a job.... this should be more complicated and use a Go channel + retry maybe 3 times + timeout.

Thoughts @kadel